README.md

```diff
@@ -30,6 +30,15 @@ the latter you will of course need MPI installed.
 The `finite_difference/example` directory contains a very simple example
 of how to construct a model grid and associate fields with it.
 
+## Runtime configuration ##
+
+The following environment variables can be set up to tune runtime options:
+
+- ALIGNMENT: Positive integer that specifies a value by which the grid contiguous
```

Comment:
On looking at this again, I think we ought to have a 'namespace' for this environment variable. Maybe call it `DL_ESM_ALIGNMENT`?

Reply:
Good idea